### PR TITLE
Delete API v1 package

### PR DESCRIPTION
**Pull request by rlucioni**
Removes old v1 views, URLs, and tests, ~~and drops the now-unused `payment_processor` field from the Order model~~. The LMS has been using v2 for about a week at this point, so killing off v1 is safe.

@clintonb and @jimabramson, could you please review this when you're able?

### CODE REVIEW

File: ecommerce/extensions/order/migrations/0008_delete_order_payment_processor.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (clintonb) [inline ecommerce/extensions/order/migrations/0008_delete_order_payment_processor.py]: Please split this into a separate PR.

**Comment by jimabramson** [comment]
@rlucioni sorry if I'm dense, but why is a view moving from v1 -> v2 before we delete it?

File: ecommerce/extensions/order/migrations/0008_delete_order_payment_processor.py (hunk 1 above)
Comment (rlucioni) [inline ecommerce/extensions/order/migrations/0008_delete_order_payment_processor.py]: @clintonb please refer to #89.

**Comment by rlucioni** [comment]
@jimabramson the view being relocated is the `OrderFulfillView`, which was [formerly imported into the v2 package](https://github.com/edx/ecommerce/pull/88/files#diff-f77c2479b2f85d8ea018d4b4a6d52081L14) in an effort to make it available at both `/api/v1/orders/fulfill/` and `/api/v2/orders/fulfill/` without duplicating code unnecessarily. Although we're deleting v1, we'd like to keep the implementation of this view in particular, which can now be moved to v2 without duplicating code.

**Comment by clintonb** [comment]
Update `acceptance_tests/config.py`.

File: ecommerce/extensions/fulfillment/mixins.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 EventHandler = get_class('order.processing', 'EventHandler')
 
 
+# TODO (RFL): Retire this mixin in favor of the `post_checkout_callback`.
```
Comment (clintonb) [inline ecommerce/extensions/fulfillment/mixins.py:10]: Ticket?

Comment (rlucioni) [inline ecommerce/extensions/fulfillment/mixins.py:10]: [XCOM-307](https://openedx.atlassian.net/browse/XCOM-307). Would you like me to include the ticket number in the comment, or were you just checking to make sure it had been created?

**Comment by rlucioni** [comment]
@clintonb is there anything in particular about `acceptance_tests/config.py` you'd like to see updated? It looks to be in good shape.

**Comment by clintonb** [comment]
My mistake. That's the enrollment API.

:+1:

**Comment by jimabramson** [comment]
:+1:
